example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -444,13 +431,29 @@ describe('The S3 Ingest Granules workflow', () => {
     let existCheck = [];
 
     beforeAll(async () => {
+      console.log('Start FailingExecution');
```

Comment:
I like this being down here because it makes it easy to `xdescribe` this whole thing and makes the test run faster when trying to track a particular issue.